app/redux_bg.ts

```diff
@@ -8,3 +8,9 @@ const createStore = store({ ...INITIAL_POPUP_STATE });
 wrapStore(createStore, {
   portName: 'INTERCEPTOR',
 })
+
+chrome.tabs.onUpdated.addListener((tabId, changeInfo, tab) => {
```

Comment:
why this is in `redux` file. suppose to be in other background page.

Reply:
I have defined two background pages. One for listening and handling our `webRequest`'s and the other for initiating the redux store in background. The `manifest.json` file is as below

https://github.com/code-mancers/interceptor/blob/2b92a5ae1f67b2cf3c2097558640a7676dc76909/app/manifest.json#L8

Reply:
I understand that. I am asking why this `chrome.tabs.onUpdated.addListener` done in redux background page. It should happen in other background page where all eventlisteners are handled.